README.md

```diff
@@ -14,7 +14,44 @@ This package is written for the Home Assistant [ha-tahoma](https://github.com/iM
 ## Installation
 
 ```bash
-pip install git+https://github.com/iMicknl/python-tahoma-api.git@master#tahoma_api
+pip install python-tahoma-api
```

Comment:
`pip install tahoma-api

Reply:
@tetienne `tahoma-api` is already taken :(

Reply:
Ah yes indeed, what about *pyhoma*, as suggested by @vlebourl ?